examples/tracing/README.md

```diff
@@ -0,0 +1,190 @@
+# Example
+
+In this example we deploy NGINX Gateway Fabric, a simple web application, configure NGINX Gateway Fabric
```

Comment:
I'm finding this a bit hard to follow. I wonder if we need to describe everything we are going to do in the example or if it's enough to describe the tracing piece of it.

Reply:
Yeah I struggled a bit with this example 😅 I simplified the description, but maybe I should change it completely remove the deploy app/ gateway/ HTTPRoutes? Just describe how to configure the tracing?

Reply:
Is documentation enough? Is there anything specific that would be in the example that wouldn't be in our docs?

Reply:
Yeah, that's a good point. Happy to remove it if we don't feel it adds value.

Reply:
I personally like this example because it's an easy step-by-step guide a user can follow. Also, it makes it easy for us to test tracing. 

@ciarams87 I think the new intro is much better 👍